internal/log_analysis/log_processor/destinations/s3_test.go

```
@@ -67,13 +67,15 @@ type mockParser struct {
 	mock.Mock
 }
 
-func (m *mockParser) Parse(log string) []*parsers.PantherLog {
+var _ parsers.LogParser = (*mockParser)(nil)
+
+func (m *mockParser) Parse(log string) ([]*parsers.PantherLog, error) {
 	args := m.Called(log)
 	result := args.Get(0)
 	if result == nil {
-		return nil
+		return nil, nil
```

We should return the error returned by the parser. 
Should be changed to: 
```
args := m.Called(log)
result := args.Get(0)
err := args.Error(1)
if result == nil {
  return nil, err
}
return result.([]*parsers.PantherLog),err

```

---

Fixed